scripts/local_docker2.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -x
+
+docker build -t fabric8-planner-builder -f deploy/Dockerfile.builder .
+mkdir -p runtime/dist && docker run --detach=true --name=fabric8-planner-builder --user=root --cap-add=SYS_ADMIN -t -v $(pwd)/runtime/dist:/dist:Z fabric8-planner-builder
```

Comment:
wasn't it better to have a single local_docker.sh file and just add a bash param whenever we want to execute these commands or the existing ones :( ?